ISISComputingGroup/ibex_gui
# Ticket689: Can lookup RB numbers from IBEX

To Test:
- Search RB numbers in the Experiment Details tab
- Confirm that a blank search field returns all possible RB numbers stored in the exp_data database
- Confirm that searches can be performed by name, role and date
- Confirm that selecting an RB number and pressing OK submits the number to the DAE

Note: Sending of a new RB number will be slow (~5s) without the following PRs:
- In [iocs](https://github.com/ISISComputingGroup/EPICS-ioc/pull/5)
- In [isisdae](https://github.com/ISISComputingGroup/EPICS-isisdae/pull/4)

Please merge these.
